packages/web/src/common/store/sign-out/sagas.ts

```diff
@@ -15,6 +16,7 @@ function* watchSignOut() {
   const audiusBackendInstance = yield* getContext('audiusBackendInstance')
   const localStorage = yield* getContext('localStorage')
   yield takeLatest(signOutAction.type, function* () {
+    yield put(resetAccount())
```

Comment:
was this just a bug you were seeing overall?

Reply:
yeah, present on web too, but less impactful